test/runtests.jl

```diff
@@ -1,32 +1,67 @@
-using NLPModels, OptimizationProblems, Test
+using NLPModels, NLPModelsJuMP, OptimizationProblems, Test
+
+function vio(c, l, u)
+  return max.(max.(c - u, 0), max.(-c + l, 0))
+end
+
+@test names(OptimizationProblems.ADNLPProblems) == [:ADNLPProblems]
+
+import ADNLPModels
+
+ndef = OptimizationProblems.ADNLPProblems.default_nvar
 
 # Test that every problem can be instantiated.
 for prob in names(OptimizationProblems.PureJuMP)
   prob == :PureJuMP && continue
   println(prob)
   prob_fn = eval(Meta.parse("OptimizationProblems.PureJuMP.$(prob)"))
-  prob_fn()
-end
+  model = prob_fn(ndef)
 
-@test OptimizationProblems.ADNLPProblems.problems == []
+  prob == :fminsrf2 && continue # issue because variable is a matrix!
+  prob == :hs61 && continue #because nlpmodelsjump is not working here https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/issues/84
+  prob in [:clplatea, :clplateb, :clplatec] && continue # issue because variable is a matrix!
```

Comment:
```suggestion
  prob == :hs61 && continue #because nlpmodelsjump is not working here https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/issues/84
  prob in [:clplatea, :clplateb, :clplatec, :fminsrf2] && continue # issue because variable is a matrix!
```